production/user_guide.md

```diff
@@ -81,8 +81,9 @@ Now that you have a token, you can tell Swagger to use it for your future reques
 
 In the "Value" box:
 
-* **Type the word "Bearer" followed by a space** (This step is critical!)
-* Paste your token
+* **First, you must type the word _Bearer_ followed by a space into the box. _Bearer_ is case-sensitive.**
+  * Failure to do this will result in a _401 Unauthorized_ error. 
```

Comment:
This is great.  Formatting question: later in this guide we have a slightly different format when referring to HTTP responses: `HTTP 200 Complete` and `HTTP 202 Accepted`.   Do we care, or should this be `HTTP 401 Unauthorized` instead of _401 Unauthorized_?

Reply:
GREAT call. Going to update the formatting per your suggestion - thanks David!